src/legacy/core_plugins/kibana/public/discover/doc_table/components/table_row.js

```diff
@@ -52,6 +52,7 @@ module.directive('kbnTableRow', function ($compile, $httpParamSerializer, kbnUrl
     scope: {
       columns: '=',
       filter: '=',
+      onRowClick: '=',
```

Comment:
none of the changes in this folder are necessary for this pr

Reply:
good catch, should have def been reverted, will do so.